File: sdk/javascript/test_sdk.js
import {chat} from './main.js';
async function main() {
  try {
    chat(
        'papo9292@gmail.com',
        'password1234',
        'Cuanto es 2+2',
        'math_problem_solver',
        (data) => {
          console.log(data);
        },
        (error) => {
          console.error('Error:', error);
        },
        () => {
          console.log('Stream completed');
        }
      );      
  } catch (error) {
    console.error('Error in the request:', error);
  }
}
main();

File: sdk/python/test_sdk.py
from main import llm
def recibir_chat():
    chat_instance = llm()
    chat_instance.set_credentials("papo9292@gmail.com", "password1234")
    for chunk in chat_instance.chat("hola como estas?", stream=True, model="math_problem_solver"):
        print(chunk)
    print("Stream completed")
recibir_chat()
